pkg/recovery/repair_test.go

```diff
@@ -201,7 +201,7 @@ func TestNewRepairHandler(t *testing.T) {
 
 		// invoke the chunk repair handler
 		err = repairHandler(context.Background(), &msg)
-		if err != nil && err.Error() != "storage: not found" {
+		if err != nil && err.Error() != "chunk repair: chunk not present in local store for repairing" {
```

Comment:
you can use `errors.Is()` if the error is extracted and exposed through `export_test`

Reply:
Yes, correct, lets have a proper error checking, not string validation.

Reply:
oh yeah.. sure